--- a/Makefile
+++ b/Makefile
@@ -171,8 +171,8 @@
 endif
 
 # Gpr2 tools
-build-tools: ${BUILD_LIBGPR2} coverage-instrument
-	${BUILDER} -XLIBRARY_TYPE=static -XXMLADA_BUILD=static \
+build-tools: coverage-instrument
+	${BUILDER} -XLIBRARY_TYPE=relocatable -XXMLADA_BUILD=relocatable \
 		${GPR2TOOLS} ${AP_GPR2}
 
 # Gnatcov instrumentation
@@ -212,8 +212,8 @@
 		${GPR2}
 
 install-tools: uninstall-tools
-	${INSTALLER} -XLIBRARY_TYPE=static -XXMLADA_BUILD=static \
-		--build-name=static --mode=usage ${GPR2TOOLS} ${AP_GPR2}
+	${INSTALLER} -XLIBRARY_TYPE=relocatable -XXMLADA_BUILD=relocatable \
+		--build-name=relocatable --mode=usage ${GPR2TOOLS} ${AP_GPR2}
 
 #########
 # setup #
